charts/oes/templates/secrets/opsmx-gitops-secret.yaml

```diff
@@ -3,10 +3,10 @@
 apiVersion: v1
 stringData:
   # Repo uri to fetch halyard configuration
-  gitcloneparam: https://{{ .Values.spinnaker.gitopsHalyard.repo.username }}:{{ .Values.spinnaker.gitopsHalyard.repo.token }}@github.com/{{ .Values.spinnaker.gitopsHalyard.repo.organization }}/{{ .Values.spinnaker.gitopsHalyard.repo.halRepository }}.git
+  gitcloneparam: https://{{ .Values.spinnaker.gitopsHalyard.repo.username }}:{{ .Values.spinnaker.gitopsHalyard.repo.token }}@github.com/{{ .Values.spinnaker.gitopsHalyard.repo.organization }}/{{ .Values.spinnaker.gitopsHalyard.repo.Repository }}.git
```

Comment:
Let's use repository in lower case just to maintain the standard.

Reply:
Updated it.